### ISSUE TITLE
Work hours/work days logic  problem  

### ISSUE THREAD

**Issue by pandiStefo** [LiveHelperChat/livehelperchat#944]
When using "work hours/work days logic is active" and i use: 
Hours from 14 Minutes from 0 Hours to 3 Minutes to 00  and i have checked  " Ignore users online statuses and use departments online hours " it does not work.The operators are online the working hours are ok and chat is offline.What can  I do ?

**Comment by PeopleInside** [comment]
You are not using an updated version... so is not easy to assist you if you are running an old version.
What version are you using?

**Comment by PeopleInside** [comment]
pandiStefo reports: 

let me explain again , for today if you see in screen shot that i have sent to you
wednesday
its working,
but if i change the hours
from 13 00   to 24 00
and to change it from 13 00 to 3 00 its not working anymore

**Comment by PeopleInside** [comment]
Maybe this issue is present because you set a start time on Thursday and you set the end time on an hours who goes on the next days.
The Live helper chat should maybe not allow you to do this or a different solution should be found.

**Comment by pandiStefo** [comment]
It would be better if there are more than a row for choosing hours within a day.For example 

Monday 

1. Hours from:  `<input type="number" max="24" >`  Minutes from:  `<input type="number" max="24" >`     Hours to:    `<input type="number" max="24" >`   Minutes to:   `<input type="number" max="24" >` 

2. Hours from:  `<input type="number" max="24" >`     Minutes from:  `<input type="number" max="24" >`    Hours to:   `<input type="number" max="24" >`    Minutes to:  `<input type="number" max="24" >` 

3. Hours from:  `<input type="number" max="24" >`     Minutes from:  `<input type="number" max="24" >`     Hours to:   `<input type="number" max="24" >`    Minutes to:  `<input type="number" max="24" >` 

 

Tuesday 

1. Hours from:  `<input type="number" max="24" >`  Minutes from:  `<input type="number" max="24" >`  Hours to:  `<input type="number" max="24" >`    Minutes to:   `<input type="number" max="24" >` 
2. Hours from:  `<input type="number" max="24" >`   Minutes from:   `<input type="number" max="24" >`  Hours to:   `<input type="number" max="24" >`  Minutes to:   `<input type="number" max="24" >` 
3. Hours from:   `<input type="number" max="24" >`  Minutes from:    `<input type="number" max="24" >`  Hours to:  `<input type="number" max="24" >`   Minutes to:  `<input type="number" max="24" >` 

Could it be possible @remdex  ?

**Comment by ipso** [comment]
This is next to impossible to get right and please everyone,  plus it can be fairly complex... What about making such a feature pluggable? Standardize on some data format or even a TRUE/FALSE result that can be returned from a custom file/class/function  that determines the online/offline status, and let people figure out the code themselves.

At least this way it can offer unlimited functionality, and even handle the most advanced cases with holidays and such too. Such plugins can be exchanged with their own config files or even integrated into cron to make things easier on non-programmers.

**Comment by remdex** [comment]
Yeah, it's just to complex to have it right. Because 13 till 3 is already different day. You can meanwhile just setup this logic using custom date range's department itself. But in any case this feature I do no think will be implemented ever in standard version. Or someone will be willing to implement it.

**Comment by pandiStefo** [comment]
@remdex  I am using Custom period logic its working for me. I think it will be a feature for your app having multiple ranges per day.
